src/bin/cli/subcommands/mod.rs

```diff
@@ -344,6 +344,39 @@ pub fn set(m: &ArgMatches<'_>) -> Result<(), Error> {
 }
 
 
+/// Command INVOICE
+pub fn invoice(m: &ArgMatches<'_>) -> Result<(), Error> {
+    let mut field = "invoice/number".to_owned();
```

Comment:
no reason to make this mutable, or a `String` for that matter, your borrowing it into `p.field()` in line 354 anyway. 
`let field = "invoice/number";`